docs/source/migration-two-dot.md

```diff
@@ -221,6 +221,42 @@ server.listen().then(({ url }) => {
 });
 ```
 
+<h3 id="existing-schema">Using Schema as Objects</h3>
```

Comment:
This anchor seems a bit ambiguous.  Maybe `schema-objects` or `object-notation`?